apps/kyb-app/src/main.tsx

```
@@ -13,6 +13,8 @@ import { App } from './App';
 import { Head } from './Head';
 import './i18next';
 import './index.css';
+import { initializeMonitoring } from '@/initialize-monitoring/initialize-monitoring';
```

Biome

[error] Shouldn't redeclare 'initializeMonitoring'. Consider to delete it or rename it.

'initializeMonitoring' is defined here:


(lint/suspicious/noRedeclare)